AxonDeepSeg/morphometrics/launch_morphometrics_computation.py

```diff
@@ -10,13 +10,15 @@
 import AxonDeepSeg.ads_utils as ads
 
 
-def launch_morphometrics_computation(path_img, path_prediction):
+def launch_morphometrics_computation(path_img, path_prediction, circle_approx = True):
     """
     This function is equivalent to the morphometrics_extraction notebook of AxonDeepSeg.
     It automatically performs all steps (computations, savings, displays,...) of the
     morphometrics extraction of a given sample.
     :param path_img: path of the input image (microscopy sample)
     :param path_prediction: path of the segmented image (output of AxonDeepSeg)
+    :param circle_approx: Boolean: if axon is considered circle, set to True else it is considered ellipse, and set to False.
```

Comment:
there is a *lot* of functions where the parameter `circle_approx` was added. Isn't there a better (less invasive) way to introduce that parameter without having to change the input params of all these functions?

Reply:
I agree, will see a more elgant way to do this.

Reply:
We could

1. Create a `config_morphometrics.json` file which will contain the circle_approx param, similar to `config_network.json file` which contains model parameters. The only downside I feel is that it would not make sense to add an exra config just to store one parameter. 

2. Encapsulte the functions of `compute_morpometrics.py` script in a class, which could have circle_approx as its attribute member. But for this we've to refactor this script and would include a couple of changes. 

On top of my head, I could only think of these ways to deal with this. @mathieuboudreau do you've any insights on this?

Reply:
PS: Personally, I feel the default way is better.  

@mathieuboudreau  for your reference: The functions: [evaluate_myelin_thickness_in_px](https://github.com/vs74/axondeepseg/blob/3ba42bcaff974cebf3a816ea6302c6effc81e6d6/AxonDeepSeg/morphometrics/compute_morphometrics.py#L164), [get_axon_morphometrics](https://github.com/vs74/axondeepseg/blob/3ba42bcaff974cebf3a816ea6302c6effc81e6d6/AxonDeepSeg/morphometrics/compute_morphometrics.py#L49), [draw_axon_diameter](https://github.com/vs74/axondeepseg/blob/3ba42bcaff974cebf3a816ea6302c6effc81e6d6/AxonDeepSeg/morphometrics/compute_morphometrics.py#L278), [get_aggregate_morphometrics](https://github.com/vs74/axondeepseg/blob/3ba42bcaff974cebf3a816ea6302c6effc81e6d6/AxonDeepSeg/morphometrics/compute_morphometrics.py#L338) and [launch_morphometrics_computation](https://github.com/vs74/axondeepseg/blob/3ba42bcaff974cebf3a816ea6302c6effc81e6d6/AxonDeepSeg/morphometrics/launch_morphometrics_computation.py#L13) call [get_axon_morphometrics](https://github.com/vs74/axondeepseg/blob/3ba42bcaff974cebf3a816ea6302c6effc81e6d6/AxonDeepSeg/morphometrics/compute_morphometrics.py#L49) function, which has `circle_approx` param to let the user choose between **equivalent_diameter** or **minor_axis of ellipse** to compute diameter. For this reason, I added additional parameter `circle_approx` in all these functions. Now, the question is [this](https://github.com/neuropoly/axondeepseg/pull/399#discussion_r530437803)

Reply:
that should probably be the purpose of another PR. We should move the discussion to a new issue. Let's see what @mathieuboudreau thinks first

Reply:
I'm going to reflect on this a bit and will either respond here if I have a strong opinion or we can discuss how to move forward at the next meeting.